set.mm

```diff
@@ -32689,7 +32752,7 @@ something like (wi (wceq (cv vx) (cv vy)) wph) ) into just (wcdeq vx vy
       $( Distribute conditional equality over abstraction.  (Contributed by
          Mario Carneiro, 11-Aug-2016.) $)
       cdeqab1 $p |- CondEq ( x = y -> { x | ph } = { y | ps } ) $=
-        ( cab wceq wb cdeqri cbvabv cdeqth ) ACFBDFGCDABCDABHCDEIJK $.
+        ( cab wceq nfv wb cdeqri cbvab cdeqth ) ACFBDFGCDABCDADHBCHABICDEJKL $.
```

Comment:
It is probably better to add `$d x y $.` here and remove the extra axioms. It looks like the metatheorem that this section on conditional equality aims to prove (more precisely: this section provides the basic theorems that can be used to automate proofs of any instance of that metatheorem) does not require to bundle x and y. Actually, proving all theorems in this section from ax-1--ax-9 would be a nice improvement, if you can.
Do you confirm @digama0 ?

Reply:
I would be okay with either approach here. On the one hand, it is nice to have more general theorems, but on the other hand it is also nice to use fewer axioms, and I think you are correct to assert that the proof does not need these lemmas to lack a `$d x y` in them.